sdk/python/distillation/benchmarking/evaluating_distill_models.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
@vis-yadav lets contribute evaluation to existing notebooks?

Reply:
+1, customer probably wants to run evaluation after doing the distillation through the notebook.